samples/bluetooth/mesh_demo/prj_nrf51_blenano.conf

```diff
@@ -18,16 +17,3 @@ CONFIG_BT_MESH_MODEL_GROUP_COUNT=2
 CONFIG_BT_MESH_LABEL_COUNT=0
 CONFIG_BT_MESH_PB_ADV=n
 CONFIG_BT_MESH_CFG_CLI=y
-
-#CONFIG_BT_MESH_LOW_POWER=y
-#CONFIG_BT_MESH_SELF_TEST=y
-#CONFIG_BT_MESH_IV_UPDATE_TEST=y
-#CONFIG_BT_MESH_DEBUG=y
-#CONFIG_BT_MESH_DEBUG_NET=y
-#CONFIG_BT_MESH_DEBUG_TRANS=y
-#CONFIG_BT_MESH_DEBUG_PROV=y
-#CONFIG_BT_MESH_DEBUG_BEACON=y
-#CONFIG_BT_MESH_DEBUG_CRYPTO=y
-#CONFIG_BT_MESH_DEBUG_ADV=y
-#CONFIG_BT_MESH_DEBUG_ACCESS=y
-#CONFIG_BT_DEBUG_LOG=y
```

Comment:
I wonder if these were intentionally left here so as to remind users of the existence of these potentially useful/relevant options (after all, this is a "sample").

Reply:
those commented options bitrot really fast and become irrelevant if they are not tested. If there is anything to remind the user, it should be part of the documentation. I argue this was just left over from someone debugging the sample.